File: Data Structures & Algorithms/subarray-sums-divisible-by-k/submission-2.py
class Solution:
    def subarraysDivByK(self, nums: List[int], k: int) -> int:
        subArr = []
        for i in range(len(nums)):
            current = []
            for j in range(i, len(nums)):
                current = current + [nums[j]]
                subArr.append(current)
        
        count = 0
        for arr in subArr:
            if sum(arr) % k == 0:
                count += 1
        return count

File: Data Structures & Algorithms/two-integer-sum/submission-2.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
  unordered_map<int,int> ns;

  for(int i = 0; i < nums.size(); i++){
    if(ns.count(target - nums[i])){
      return {ns[target - nums[i]], i};
    }
    ns[nums[i]] = i;
  }

  return {};


    }
};
